components/GuideTemplate/Menu/Menu.js

```diff
@@ -8,10 +8,12 @@ import './Menu.scss';
 
 class Menu extends Component {
   _renderSubMenuItem(subMenuItemContent, index) {
-    const { activeSubtitle } = this.props;
-    const isActive =
-      activeSubtitle &&
-      subMenuItemContent.title === activeSubtitle;
+    let isActive = false;
+    if (window.location.hash) {
+      isActive = subMenuItemContent.url === window.location.hash;
```

Comment:
Can we use `router` state instead of window.location?

I remember react-router provide current url (Not sure)